src/FrameBuffer.cpp

```diff
@@ -1369,15 +1380,25 @@ void FrameBufferToRDRAM::_copy(u32 _startAddress, u32 _endAddress, bool _sync)
 	if (m_pCurFrameBuffer->m_size == G_IM_SIZ_32b) {
 		u32 *ptr_src = (u32*)pixelData;
 		u32 *ptr_dst = (u32*)(RDRAM + _startAddress);
-		_writeToRdram<u32, u32>(ptr_src, ptr_dst, &FrameBufferToRDRAM::_RGBAtoRGBA32, 0, 0, width, height, numPixels, _startAddress, m_pCurFrameBuffer->m_startAddress, m_pCurFrameBuffer->m_size);
+		u32 srcBuf[width * height];
```

Comment:
Only g++ can do that trick. MSVS gets error:
error C2131: expression did not evaluate to a constant

If you don't want to mess with new/delete, you may do like this:
```
std::vector<u32> srcBuf(width * height);
u32 * pSrc = srcBuf.data();
memcpy(pSrc, ptr_src, width * height*sizeof(u32));
....
```